### ISSUE TITLE
The prediction seems not correct

### ISSUE THREAD

**Issue by luzai** [wangnamu/AICamera_new#1]
Thanks a lot for release the pre-build caffe2 lib for android! 

I test the code on Huawei Tag-AL00 (Android 5.1, API 22), the prediction seems not correct (Shown in the last). Could you help me to reproduce the results? 

Meanwhile, I read some related issues. May I ask why the input to the network seems to be of range (0,255), rather than (0,1), as shown in [here](https://github.com/wangnamu/AICamera_new/blob/da47cb40aa3473dabb484c01b98d32b485b9d9fc/AICamera/app/src/main/cpp/native-lib.cpp#L129). 

![image](https://user-images.githubusercontent.com/14788650/50442832-a8218400-093b-11e9-91f1-ff9d122b125f.png)

**Comment by wangnamu** [comment]
hi @luzai 

Maybe you should read [this](https://github.com/caffe2/AICamera/blob/master/Exporting%20Squeezenet%20to%20mobile.ipynb) first, input from range(0,255) because the input of the training set is such a format, and you can train your own model input range(0,1) and convert to caffe2 through pytorch.

I have not tested it on the Huawei phone, but the result is not correct. I suspect that it is a problem of yuv conversion rgb. For different android devices, the direction of the image read from the camera will be different. I suggest you try the library with libyuv. Convert to rgb format and check if the converted data is correct. Alternatively, you can try to use a bitmap to read the image in your phone and then convert it to rgb format to verify the accuracy of the model.

I hope it helps you  :)

**Comment by luzai** [comment]
Thank you very much for your suggestions! 

I try to read the image in my phone, use the default image processor of pytorch and the result is correct! I am still wondering about the input format. It seems that no matter pytorch model or converted caffe2 model need RGB input from range(-2,2) approximately. As shown below, `torch_model` is loaded from `*.pth` and `p` is loaded from `*.pb`:
![image](https://user-images.githubusercontent.com/14788650/50516509-26218e80-0ae6-11e9-8cd1-f9f6b3908969.png)

![image](https://user-images.githubusercontent.com/14788650/50516527-3e91a900-0ae6-11e9-8680-46d1965ca8df.png)

**Comment by wangnamu** [comment]
hi @luzai,I updated the code and processed the image with libyuv. I still haven't tested it on Huawei phones, but I used some other mobile tests and the results are correct. You can try again.  
I hope it helps you :)

**Comment by luzai** [comment]
Great thanks for your demonstration on processing the image with libyuv! I test on Huawei phones, the results are correct. Thank you! :)

**Comment by zhengye1995** [comment]
@luzai so you trained your own model with the input range（0,255）? I meet the same problem.
